### PR TITLE
:bulb: add README

### PR DESCRIPTION
**Pull request by namenyi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,66 @@
-# nacelle-nuxt-module
-Recart integration for Nacelle
+# Recart integration for Nacelle
+
+Recart works out of the box on normal Shopify storefronts. 
+
+However, if you have a Nacelle site, you need to install this Nuxt module make Recart work.
```
Comment (gabor-recart) [inline README.md]:
```suggestion
However, if you have a Nacelle site, you need to install this Nuxt module to make Recart work.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,66 @@
...
+
+    Open `nuxt.config.js` and add `@recart/nacelle-nuxt-module` to the `modules` array. It should look like this:
+
+    ```
+    modules: [
+      '@nuxtjs/pwa',
+      '@nuxtjs/dotenv',
+      '@nacelle/nacelle-nuxt-module',
+      '@nuxtjs/sitemap',
+      '@recart/nacelle-nuxt-module'
```
Comment (gabor-recart) [inline README.md]:
```suggestion
      ['@recart/nacelle-nuxt-module', { recartSiteId: '5abcdefgh000000000000000' }]
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,66 @@
...
+
+    ```
+    nacelle: {
+      spaceID: process.env.NACELLE_SPACE_ID,
+      token: process.env.NACELLE_GRAPHQL_TOKEN,
+      gaID: process.env.NACELLE_GA_ID,
+      fbID: process.env.NACELLE_FB_ID,
+      recartSiteId: '5abcdefgh000000000000000'
+    },
+    ```
```
Comment (gabor-recart) [inline README.md]: ezt lehet explicit a modulenak adni, a fenti commit suggestionben leirtam hogy

Comment (namenyi) [reply README.md]: cool, I didn't know that, thanks! Will fix the description soon

**Review by gabor-recart — APPROVED** [review]

**Pull request merged**